docs/deployment.rst

```diff
@@ -145,18 +142,23 @@ Kafka & ZooKeeper
 
 For recommendations on operationalizing Kafka and |zk|, see :ref:`schemaregistry_operations`.
 
+.. _schemaregistry_zk_migration:
+
 Migration from ZooKeeper master election to Kafka master election
 -----------------------------------------------------------------
 
-It is not required to migrate from |zk| based election to Kafka based master election. If
-you choose to do so, you need to make the below outlined config changes as the first step in all
-|sr| nodes
+|zk|-based master election is available in all versions of |sr|, and if you have an existing |sr| deployment you may continue to use it for compatibility. It is not required to migrate from |zk| based election to Kafka based master election.
+
+Kafka-based master election can be used in version 4.0 and later in cases where |zk| is not available, for example for hosted or cloud Kafka environments, or if access to |zk| has been locked down. If you choose to migrate from |zk|-based to Kafka-based master election, make the following configuration changes in all |sr| nodes:
```

Comment:
suggest:

```
You can use Kafka-based master election in cases where |zk| is not available. 
For example, on hosted or cloud environments, or if access to |zk| has been locked down. 
To migrate from |zk|-based to Kafka-based master election, make the following configuration 
changes in all |sr| nodes:
```

Reply:
I made a few changes here in the last commit